Makefile

```diff
@@ -260,8 +270,11 @@ predeploy-aws-account-operator: ## Predeploy AWS Account Operator
 	# Create zero size account pool
 	@oc apply -f hack/files/aws.managed.openshift.io_v1alpha1_zero_size_accountpool.yaml
 
+.PHONY: validate-deployment
+validate-deployment: check-aws-account-id-env check-sts-setup ## Validates deployment configuration
+
 .PHONY: predeploy
-predeploy: predeploy-aws-account-operator deploy-aws-account-operator-credentials create-ou-map ## Predeploy Operator
+predeploy: predeploy-aws-account-operator deploy-aws-account-operator-credentials create-ou-map validate-deployment ## Predeploy Operator
```

Comment:
Should validate-deployment be moved to the beginning of the list as we'll want to validate this info before continuing with the deployment